### ISSUE TITLE
youtube does not scrobble

### ISSUE THREAD

**Issue by ilmailma** [web-scrobbler/web-scrobbler#2669]
today during the day i noticed that i can't get the plugin to scrobble on youtube, no music was sent to last.fm. the plugin shows as if it is available on the page, but does not detect any music. would be great if you managed to find a way to fix it, please :) thanks!   ![image](https://user-images.githubusercontent.com/15608601/102536865-7c5cb300-40aa-11eb-9f81-e1558a16753b.png)

**Comment by inverse** [comment]
@ilmailma could you provide more information.

- link to track that's failing to scrobble
- provide browser info
- debug logs (check wiki)

**Comment by Lubadubs** [comment]
Hello,

I encountered the same issue so I thought I might help with the logs. I hope I made the logs correctly.

I am using Google Chrome version 87.0.4280.88.

"Works" for every song on YouTube (I tried some), but here the link to the one I used for the logs: https://www.youtube.com/watch?v=0XpSr9BSYr4

Debug log webscrobbler: https://pastebin.com/wPQEzXmH
Debug log website: https://pastebin.com/RpzBva7S

Thank you already in advance!
